docs/tips.rst

```diff
@@ -246,7 +246,52 @@ Here's the complete list of workflow inputs:
 |                  |          | your GitHub Pages site, which is correct   |        |               |
 |                  |          | in most cases.                             |        |               |
 +------------------+----------+--------------------------------------------+--------+---------------+
+| ``deploy``       | No       | This is used to determine whether you will | bool   | ``true``      |
+|                  |          | deploy the site or not to GitHub Pages.    |        |               |
+|                  |          | This is most useful if you want to test a  |        |               |
+|                  |          | change to your website in a pull request   |        |               |
+|                  |          | before deploying those change.             |        |               |
++------------------+----------+--------------------------------------------+--------+---------------+
+
+Testing your build in a GitHub Pull Request
```

Comment:
I would capitalize "Your" and "Build", to be consistent with (most) of the headers in this file.